infrastructure/cdn-in-a-box/to_integration/Dockerfile

```diff
@@ -0,0 +1,55 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+FROM golang:1.11 AS integration-builder
+
+# integration source and dependencies
+COPY ./lib/ /go/src/github.com/apache/trafficcontrol/lib/
+COPY ./vendor/ /go/src/github.com/apache/trafficcontrol/vendor/
+COPY ./traffic_ops/client/ /go/src/github.com/apache/trafficcontrol/traffic_ops/client/
+COPY ./traffic_ops/vendor/ /go/src/github.com/apache/trafficcontrol/traffic_ops/vendor/
+COPY ./infrastructure/cdn-in-a-box/to_integration/ /go/src/github.com/apache/trafficcontrol/infrastructure/cdn-in-a-box/to_integration/
+COPY ./traffic_ops/testing/api /go/src/github.com/apache/trafficcontrol/traffic_ops/testing/api
+COPY ./traffic_ops/traffic_ops_golang /go/src/github.com/apache/trafficcontrol/traffic_ops/traffic_ops_golang
+COPY ./infrastructure/cdn-in-a-box/ /go/src/github.com/apache/trafficcontrol/infrastructure/cdn-in-a-box/
+
+# if we end up using a different versioning convention, the compile command will need to change
+WORKDIR /go/src/github.com/apache/trafficcontrol/traffic_ops/testing/api
+RUN go get -u golang.org/x/net/publicsuffix golang.org/x/crypto/scrypt
+RUN go test -c ./v1* -o traffic_ops_integration
+
+FROM debian:stretch
+# Based off enroller. Might be able to remove some things from the install.
```

Comment:
The best way to check is to remove the things you think this container might not need and see if it still builds/runs the tests.

Reply:
It still runs the tests without netcat, dnsutils, and net-tools but it has different output. To be safe I'll keep them included. I've built the container with curl and [netcat|dnsutils|net-tools], all with similar responses. 

For each it says something along the lines of:
`Suggested packages:`
`The following NEW packages will be installed:` or
`The following additional packages will be installed:`

It does not do this if all three are included in the build.